index.html

```diff
@@ -56,7 +58,7 @@ <h1 class="primary1">I'm Selma, <br> Glad to see you</h1>
   
 
     <!--Works Section-->
-    <section class="works">
+    <section class="works" id="Portfolio">
```

Comment:
- [ ] [optional] I noticed your work section has a fixed width, this makes it unresponsive kindly remove the fixed width so that the design is good in different screen sizes.